sys/base64/Kconfig

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_BASE64
+    bool "Base64 codec"
+    depends on TEST_KCONFIG
+
+config MODULE_BASE64URL
+    bool "Base64 URL codec"
+    select MODULE_BASE64
```

Comment:
If you choose to `select`, then `MODULE_BASE64URL` `depends on TEST_KCONFIG` as well